routes/discover.js

```diff
@@ -1,8 +1,11 @@
 var data = require('../testingOrganizations.json');
 
-exports.view = function(req, res) {
+exports.mapsKey = function(req, res) {
```

Comment:
nit: Renaming to `mapsKeys` is confusing to me. `view` was a fine name or is there a reason for the rename.

Reply:
mapsKey just returns the Google Maps API key.

View was used previously since it returned the key and organization data.

Reply:
Oh I see but the method is still rendering the 'discover' view that is at `views/discover.handlbars` like last time. Its now just passing in only `MAPS_KEY` **into** the handlebars file. It's not returning the Maps API key itself but its returning/rendering the view generated by the handlebars file.

Reply:
Oh gotcha. That makes sense. Changed it back 🙂